### PR TITLE
Sets up starships component, displays list of ships

### PR DESCRIPTION
**Pull request by brandon-fp**
Set up display of all starships (just 10 display)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,26 @@
...
+	componentWillMount(){
+    
+	}
+
+	componentDidMount(){
+	}
+
+	componentWillReceiveProps(){
+
+	}
```
Comment (czaas) [inline src/App.js]: Remove the empty componentWillMount, componentDidMount, and componentWillReceiveProps

**Review by czaas — CHANGES_REQUESTED** [review]
Just a couple empty methods that need to be removed and this will be good to go.

File: src/components/Starships/index.js [hunk 2]
```diff
@@ -1,16 +1,99 @@
...
+  }
+
   componentDidMount() {
-    fetchShips();
+    const that = this;
+
+//    do {
+      swapi.getStarships(that.state.page, res => {                
+        var next = that.getUrlParams(res.next)['page'];      
+        that.setState({
```
Comment (czaas) [inline src/components/Starships/index.js]: I like where you're at on this. Just FYI: You can update multiple properties of the state in the same setState call. 

Move the page and ships updates in the same setState call.

File: src/components/Starships/index.js [hunk 3]
```diff
@@ -1,16 +1,99 @@
...
+
   }
+
+  updateSearch = (e) => {
+    this.setState({
+      currentSearch: e.target.value
+    })
+  }
+
+  getUrlParams = (url) => {
```
Comment (czaas) [inline src/components/Starships/index.js]: I think this will end up getting used in most of the files. Let's talk about moving this into a utility file.

**Comment by brandon-fp** [comment]
Completed. Also changed "var" to "let" for next variable. Do I click "dismiss review" or take no further action?

**Review by czaas — APPROVED** [review]

**Pull request merged**